ServerCore/Pages/Resources/nipd2023/HomePartial.cshtml

```diff
@@ -0,0 +1,54 @@
+                <div class="column-one-third">
+                    <h3>WELCOME TO NIPD</h3>
+                    <p>Non-Intern Puzzleday is a fun 6-hour event where teams of up to 6 players solve puzzles. We accept puzzle solvers of all levels! Check out our <b><a asp-page="/EventSpecific/FAQ">FAQ</a></b> and <b><a asp-page="/EventSpecific/Rules">Rules</a></b> pages for more information.</p>
+                </div>
+                <div class="column-two-third">
+                    <img src="~/images/pd2023/nipd2023logo.png" height="160" width="160" title="Puzzleday 2023 logo" alt="Puzzleday 2023 logo" />
+                </div>
+             </div>
+
+            <div class="pd2023-container-card card-2">
+                <h3>NEW TO PUZZLES?</h3>
+                <p>You may have seen a jigsaw puzzle, a crossword puzzle, or a Sudoku at some point in your life, but those barely crack the cover on the whole anthology that is &quot;puzzling&quot;! Puzzleday puzzles also have a tendency to get... creative, so you might see something that looks especially strange! Check out a few <a asp-page="/EventSpecific/Samples">sample puzzles</a> or peruse our <a href="https://puzzlearchive.azurewebsites.net/puzzleday/main.html" target="_blank">collection of puzzles</a> from past events.</p>
+            </div>
+
+            <div class="pd2023-container-card card-3">
+                <h3>FULL SCHEDULE (all times PDT)</h3>
+                <p>This year's event will take place on <b>August 26<sup>th</sup></b>. Additionally, pregame puzzles will be available in the days leading up to the event.  On the day of:</p>
+                <ul>
+                    <li><b>11AM:</b> Opening ceremony in Building 99, followed by release of puzzles.</li>
+                    <li>During the day, live puzzles will be running in Building 99. Look at those puzzles for the specific time and place.</li>
+                    <li><b>5PM:</b> Puzzle submission closes.</li>
+                    <li><b>5:10PM:</b> Brief wrap-up in Building 99.</li>
+                </ul>
+            </div>
+
+            <div class="pd2023-container-card card-4">
+                <h3>HOW TO SIGN UP</h3>
+                <p>NEW FOR THIS YEAR: Before creating or joining a team, every player must register for this specific event using the Register link at the top of this page.  During this process, you will select whether you will be in-person or remote.  Once you're registered, you will be able to solve Pregame puzzles once they're available, even if you're not on a team yet! Remote participants will not be able to play in the live puzzles, but they will be able to finish the event.</p>
+                <p>If you don't have a team yet, you can <b><a asp-page="/Teams/List">create a team using the link here</a></b>. At least 2 members of every team must be Microsoft FTEs. Guests on campus must be accompanied by an FTE at all times, and interns are not allowed to play in this event!</p>
```

Comment:
@asyasky Does this link need to go somewhere else or be conditional on registration?

Reply:
Done!